web/index.css

```diff
@@ -416,3 +380,31 @@ div.toggle {
   padding: 4px;
   background: #3f3f3f;
 }
+
+.sharing-dialog {
+  top: 100px;
+  left: 100px;
+  right: 100px;
+}
```

Comment:
agree it's too wide; I like your style changes. I'll make this the default for dialogs.